tools/vendor.sh

```diff
@@ -0,0 +1,73 @@
+set -e
```

Comment:
missed this earlier - might need a shebang line to make this execute w/ bash

Reply:
Done!